po/pt_BR.po

```diff
@@ -0,0 +1,1657 @@
+# Code generated by tools/update-translator-sources. DO NOT EDIT.
+# source: (various)
```

Comment:
Must fix: This comment shouldn't exist. (I should update the instructions to remove this message...)